docs/apache-airflow/deprecated-rest-api-ref.rst

```diff
@@ -31,6 +28,9 @@ available at ``/api/experimental/``.
   The experimental REST API is disabled by default. To restore these APIs while migrating to
   the stable REST API, set ``enable_experimental_api`` option in ``[api]`` section to ``True``.
 
+Airflow exposes an REST API. It is available through the webserver. Endpoints are
+available at ``/api/experimental/``.
```

Comment:
```suggestion
Before Airflow 2.0 this REST API was knwon as the "experimental" API, but now that the :ref:`stable REST API <stable-rest-api-ref>` is available, it has been renamed.

The endpoints for this API are available at ``/api/experimental/``.
```

What do you think of this @uranusjr ?

Reply:
LGTM. How about also edit the `.. versionchanged:: 2.0` block above. It refers to this API as experiemental, and it would make a lot of sense if this new paragraph appears first to explain why that word is used.

Reply:
Yup, noticed that when editing it locally. Will do.